gen2-nn-sync/requirements.txt

```diff
@@ -0,0 +1,4 @@
+opencv-python==4.2.0.34; platform_machine != "armv7l"
```

Comment:
Use `opencv-python==4.5.1.48` for all platforms.
Same in other examples (if applicable)

Reply:
Sure, changed and tested in this example and also updated others